lib/presentation/ui/home/home_page.dart

```diff
@@ -45,11 +45,11 @@ class _MyHomePageState extends BaseState<HomePage> {
     return Scaffold(
       key: scaffoldKey,
       body: StreamBuilder<int>(
-          initialData: widget.bloc.currentPageSubject.value,
-          stream: widget.bloc.currentPageStream,
-          builder: (context, snapshot) {
-            return _pages[snapshot.data];
-          }),
+            initialData: widget.bloc.currentPageSubject.value,
```

Comment:
Does this value can never be `null`? If so, it should be null aware protected.

Reply:
Nope, never:
 `var currentPageSubject = BehaviorSubject<int>(seedValue: 0);`